src/lib/draw/point/point_tool.ts

```diff
@@ -19,7 +18,22 @@ export class PointTool {
   eventListenersFailure: (() => void)[];
   cursor: FeatureWithProps<Point> | null;
 
-  events: EventManager;
+  onMouseMove = (e: MapMouseEvent) => {
+    if (this.active) {
```

Comment:
I think we can remove the `this.active` checks now? We're only forwarding events to the tools when they're in use

Reply:
We shall see. Currently geometry mode is misbehaving weirdly so I don't wanna jump the gun on this. I've gotten rid of a lot of `currentMode` checks but I think geom mode means `this.active` checks probably wanna stay

Reply:
I might've been wrong here! We still need the currentMode checks in the Svelte components